bangazonapi/views/paymenttype.py

```diff
@@ -81,10 +81,10 @@ def list(self, request):
         """Handle GET requests to payment type resource"""
         payment_types = Payment.objects.all()
 
-        customer_id = self.request.query_params.get('customer', None)
+        customer_id = self.request.query_params.get(request.auth.user)
```

Comment:
Since `query_params` are in the url after a`?` it would be better to get the customer based on the logged in user instead